whiskey.yml

```diff
@@ -3,6 +3,7 @@ Build:
 - Version:
     Path: Carbon\Carbon.psd1
     Prerelease:
+    - main: ""
     - master: ""

```

Comment:
master can probably be removed since we changed that to main?

Reply:
Good catch.